pom.xml

```diff
@@ -39,27 +38,15 @@
       <artifactId>github-api</artifactId>
       <version>1.67</version>
     </dependency>
-    <dependency>
+    <dependency><!-- minimun required version to work with Workflow 1.4 GitSCM step-->
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>git</artifactId>
-      <version>2.0</version>
-    </dependency>
-    <dependency>
-      <groupId>org.jmock</groupId>
-      <artifactId>jmock-junit4</artifactId>
-      <version>2.5.1</version>
-      <scope>test</scope>
+      <version>2.3.5</version>
     </dependency>
     <dependency>
-      <groupId>org.eclipse.jgit</groupId>
-      <artifactId>org.eclipse.jgit</artifactId>
-      <version>0.12.1</version>
-    </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.plugins</groupId>
-      <artifactId>multiple-scms</artifactId>
-      <version>0.2</version>
-      <optional>true</optional>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.6.6</version>
```

Comment:
the slf-api (and slf4j-jdk14) are already provided by jenkins core (since a long time ago), so this should probably be marked as `<scope>provided</scope>`